src/compas_robots/blender/scene/robotmodelobject.py

```diff
@@ -1,44 +1,39 @@
-from typing import Union
-from typing import Tuple
-from typing import Optional
 from typing import Any
 from typing import List
+from typing import Optional
+from typing import Union
 
 import bpy  # type: ignore
+import compas_blender
 import mathutils
-
+from compas.colors import Color
 from compas.datastructures import Mesh
 from compas.geometry import Transformation
+from compas_blender.scene import BlenderSceneObject
 
-import compas_blender
-from compas_blender.artists import BlenderArtist
-from compas_blender.utilities import RGBColor
-
-from compas_robots.artists import RobotModelArtist
-from compas_robots.robots import RobotModel
+from compas_robots import RobotModel
+from compas_robots.scene import BaseRobotModelObject
 
 
-class RobotModelArtist(BlenderArtist, RobotModelArtist):
-    """Artist for drawing robot models in Blender.
+class RobotModelObject(BlenderSceneObject, BaseRobotModelObject):
```

Comment:
mmh `RobotModelObject` sounds a bit too generic to me `RobotModelSceneObject` too long/verbose?

Reply:
I had it like that at first, but the convention in core seems to be `XxxObject` not `XxxSceneObject` (eg `BoxObject`, `NetworkObject`, etc)